package.json

```diff
@@ -63,6 +63,7 @@
     "minimist": "^1.2.0",
     "mocha": "^5.2.0",
     "sinon": "^4.5.0",
+    "ssl-root-cas": "^1.3.1",
```

Comment:
is there a way to avoid adding this dependency?

Reply:
I figured out a way, it's gone now